velox/exec/Operator.h

```diff
@@ -343,14 +356,20 @@ class Operator {
       int32_t id,
       const std::shared_ptr<const core::PlanNode>& planNode);
 
+  // Calls all the registered PlanNodeTranslators on 'planNode' and
+  // returns the result of the first one that returns non-nullptr
+  // or nullptr if all return nullptr.
+  static OperatorSupplier operatorSupplierFromPlanNode(
+      const std::shared_ptr<const core::PlanNode>& planNode);
+
   // Calls `maxDrivers` on all the registered PlanNodeTranslators and returns
   // the first one that is not std::nullopt or std::nullopt otherwise.
   static std::optional<uint32_t> maxDrivers(
       const std::shared_ptr<const core::PlanNode>& planNode);
 
- protected:
   static std::vector<std::unique_ptr<PlanNodeTranslator>>& translators();
 
+ protected:
```

Comment:
Is this change needed? Can translators() method stay protected?

Reply:
add joinBridgeFromPlanNode method in Operator to solve it.